_docs/vision.md

```diff
@@ -13,51 +13,40 @@ to present itself as more than just one database? Why not create software which
 more important thing. In fact, migrating data to PostgreSQL can be relatively
-easy, while migration entire application stacks can be really difficult. 
+easy, while migrating entire application stacks can be really difficult.
 
 That's exactly why Babelfish is so important. It allows you to take existing
 applications, hook them up to TDS-enabled PostgreSQL and run them more quickly
 and more easily, even while some migration work might still be going on. 
 
-A second option is to move application from MS SQL to PostgreSQL without
-touching the application at all. Why is that so important? If you get your
-source code under control, life is usually easier. But what if you want to move
-old legacy applications from Microsoft SQL Server to PostgreSQL, legacy applications which cannot be
-modified easily, because the source code is not open, is lost, or is simply too hard to
-change? Babelfish is an excellent vehicle to help in this case as well. You can
-move away from expensive MS SQL deployments without doing a painful app
-migration. Brilliant, isn't it?
+If you are stuck with a legacy application, where the source code is not
+available or too hard to modify for technical or financial reasons, Babelfish
+can provide an easy way out.  The flexibility and cost savings offered by
+open source software projects like PostgreSQL with Babelfish can be significant.
 
 
 ### What Babelfish is not
 
-While Babelfish can ease a lot of the burden MS SQL users experience, it is not a drop-in
+While Babelfish can ease a lot of the burden Microsoft SQL Server users experience, it is not a drop-in
 replacement for everything. There is code (especially some server-side 
 procedures related to administration) that is not going
 to work on Babelfish. The reason is that every database engine has limitations.
 The same is of course true for PostgreSQL and therefore some exotic features cannot be mapped. 
 However, we do our best to keep this list as short as humanly possible.
 
 If you want to find out more about limitations: We have dedicated an entire
-[section dealing with limitations incompatibilities](/docs/usage/limitations-of-babelfish).
+section to the [limitations](/docs/usage/limitations-of-babelfish).
 
-### Preparing for the future
-
-Nothing is perfect, but the community is working hard to make things
-function as smoothly as possible. Much of it works already, but more 
-work has to go into this area to be even more compatible. In other words: 
-We have an exciting future ahead. Many more features will be developed as 
-people improve the user experience, raising it to new levels. 
-
-Be part of the community and help to improve Babelfish even more.
-If you want to [contribute to Babelfish](/docs/contributing) simply reach
-out to the community.
+### Join the Babelfish community
 
+Be part of the community and help to improve Babelfish.
+If you want to [contribute to Babelfish](/docs/contributing), you are welcome.
```

Comment:
correction:   "you're welcome to." 
-->alternative: "We welcome [contributions to Babelfish](/docs/contributing)

Reply:
Done.